addons/test_mail/tests/test_message_post.py

```diff
@@ -26,6 +26,11 @@ def setUpClass(cls):
         cls._reset_mail_context(cls.test_record)
         cls.user_admin.write({'notification_type': 'email'})
 
+        cls.test_track = cls.env['mail.test.track'].with_context(cls._test_context).with_user(cls.user_employee).create({
```

Comment:
Out of curiosity, why not doing the test on "mail test simple" (cls.test_record) define hereunder ? Any issue with it ? Otherwise let us keep test data simple and keep a single record for all tests if possible :)

Reply:
The template needs the record to have a user_id and the test_record doesn't have one.

Reply:
Ok then, maybe for that case we could define it only in the test. That way global class variables are kept as small as possible, if it is used only once let us define it only in that new test.